addon/components/tag-input.hbs

```diff
@@ -1,5 +1,5 @@
 {{#each this.tags as |tag index|~}}
-  <li class="emberTagInput-tag">
+  <li class="emberTagInput-tag {{tag.modifiers}}">
```

Comment:
I'm wondering if the term "modifiers" is confusing here when it is really meant to hold custom classnames 🤔 

With he current name, I would almost expect it to represent an array of ember-modifier methods or something.

Reply:
I guess the distinction is that this is a classname modifier vs. an element modifier.

Reply:
yeah, I'm not a huge fan of it myself - but with not changing that this is really just a refresh of an existing PR with none of my opinion bleeding into it.

I probably would have gone with `classes`

Reply:
I guess if the PR never gets accepted we can come back and play more freely